docs/docs/migration/migration-from-v6-to-v7.md

```diff
@@ -831,6 +831,53 @@ The `InternalLinkBlock` provided by `@comet/cms-site` is deprecated.
 Instead, implement your own `InternalLinkBlock`.
 This is needed for more flexibility, e.g., support for internationalized routing.
 
+### Add `legacyBevavior` to all link block usages
+
+Previously, Next required the `Link` component to have a child `<a>` tag. To style this tag correctly in the application, none of the library link blocks rendered the tag, but cloned the children with the correct props instead. However, since Next v13 the `Link` component no longer requires a child `<a>` tag. Consequently, we don't need to render the tag for the `InternalLinkBlock` (which uses `Link` internally) anymore. In order to style all link blocks correctly, we now render an `<a>` tag for all other link blocks.
```

Comment:
that paragraph describes the situation without legacyBehaviour but in the migration guide we advise to use the legacyBehaviour

Reply:
I wanted to give some context why this change is necessary. But I'll rewrite it to make it more specific.

Reply:
Changed: fc7aa4e95f07d7e25d94be477b586c37258164fe